packages/shared/src/components/buttons/ToggleClickbaitShield.tsx

```diff
@@ -10,16 +11,20 @@ import { useLazyModal } from '../../hooks/useLazyModal';
 import { SimpleTooltip } from '../tooltips';
 import { LazyModal } from '../modals/common/types';
 import { FilterMenuTitle } from '../filters/helpers';
+import { ActiveFeedContext } from '../../contexts';
 
 export const ToggleClickbaitShield = ({
   origin,
 }: {
   origin: Origin;
 }): ReactElement => {
+  const queryClient = useQueryClient();
+  const { queryKey: feedQueryKey } = useContext(ActiveFeedContext);
```

Comment:
```suggestion
  const { queryKey: feedQueryKey } = useActiveFeedNameContext();
```

Reply:
Oh there's a separate context for feed name? 😬

Reply:
ActiveFeedNameContext only gets feed name, not the query key

Reply:
Oh sorry no I meant is there not a hook export instead of using `useContext` used the wrong one here.

Reply:
Nope, no hook for that one